sui/tokens.js

```diff
@@ -43,48 +43,23 @@ class CoinManager {
     }
 
     static async splitCoins(tx, coinTypeToCoins, options) {
-        CoinManager.checkSplitAmount(options);
-        const splitAmount = BigInt(options.split);
-
-        console.log('\n==== Splitting Coins ====');
+        printInfo('\n==== Splitting Coins ====');
```

Comment:
print the action name in `processCommand` instead of printing in each helper function. For consistency, use `printInfo('Action', actionType)` print format